addons/default/visiosoft/profile-module/src/Profile/Register2/Register2FormFields.php

```diff
@@ -73,6 +73,13 @@ public function handle(Register2FormBuilder $builder)
             array_merge(
                 $register,
                 [
+                    "register_type" => [
+                        "type" => "anomaly.field_type.select",
+                        "config" => [
+                            "options" => ['personal' => 'visiosoft.module.profile::field.personal.name', 'corporate' => 'visiosoft.module.profile::field.corporate.name'],
+                            "mode" => "radio",
+                        ],
+                    ],
```

Comment:
what happens if there is no store module? This should only come if the store module is installed